plugins/core/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -394,6 +394,7 @@ caws.workspace.details.branch_new_validation=New branch name is required
 caws.workspace.details.branch_title=Branch:
 caws.workspace.details.branch_validation=Branch selection is required
 caws.workspace.details.clone_repo=Clone a repository
+caws.workspace.details.code_gen_iterations=You have {0} out of {1} code iterations remaining.
```

Comment:
```suggestion
amazonqFeatureDev.code_gen_iterations=You have {0} out of {1} code iterations remaining.
```

Reply:
Ok. Updated to"
`amazonqFeatureDev.code_generation.iteration_counts=You have {0} out of {1} code iterations remaining.`